### PR TITLE
10.0 mig stock_picking_invoice_link

### PR DESCRIPTION
**Pull request by eLBati**
Based on https://github.com/OCA/stock-logistics-workflow/pull/296
This is WIP, waiting for the destiny of https://github.com/OCA/stock-logistics-workflow/pull/296

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please squash some commits together (like "OCA Transbot" ones), and take into account https://github.com/OCA/stock-logistics-workflow/pull/296

**Comment by eLBati** [comment]
Hi @pedrobaeza , squashed some commits.
Also, this PR is already based on https://github.com/OCA/stock-logistics-workflow/pull/296 and, since 10 april, no commits were added

File: stock_picking_invoice_link/views/account_invoice_view.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
```
Comment (grindtildeath) [inline stock_picking_invoice_link/views/account_invoice_view.xml:2]: odoo

File: stock_picking_invoice_link/views/account_invoice_view.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    <data>
```
Comment (grindtildeath) [inline stock_picking_invoice_link/views/account_invoice_view.xml:3]: To remove

File: stock_picking_invoice_link/views/stock_view.xml [hunk 3]
```diff
@@ -1,16 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
 <openerp>
```
Comment (grindtildeath) [inline stock_picking_invoice_link/views/stock_view.xml:2]: odoo

File: stock_picking_invoice_link/views/stock_view.xml [hunk 4]
```diff
@@ -1,16 +1,19 @@
...
-<data>
+    <data>
```
Comment (grindtildeath) [inline stock_picking_invoice_link/views/stock_view.xml:3]: To remove

File: stock_picking_invoice_link/tests/test_stock_picking_invoice_link.py [hunk 5]
```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+# © 2016 Oihane Crucelaegui - AvanzOSC
+# © 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# © 2017 Jacques-Etienne Baudoux <je@bcim.be>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp.addons.sale.tests.test_sale_common import TestSale
```
Comment (grindtildeath) [inline stock_picking_invoice_link/tests/test_stock_picking_invoice_link.py:7]: odoo.addons

File: stock_picking_invoice_link/models/stock_picking.py [hunk 6]
```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# © 2013-15 Agile Business Group sagl (<http://www.agilebg.com>)
+# © 2015-2016 AvanzOSC
+# © 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# © 2017 Jacques-Etienne Baudoux <je@bcim.be>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import api, fields, models
```
Comment (grindtildeath) [inline stock_picking_invoice_link/models/stock_picking.py:8]: from odoo

File: stock_picking_invoice_link/models/stock_move.py [hunk 7]
```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# © 2013-15 Agile Business Group sagl (<http://www.agilebg.com>)
+# © 2015-2016 AvanzOSC
+# © 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import api, fields, models
```
Comment (grindtildeath) [inline stock_picking_invoice_link/models/stock_move.py:7]: from odoo

File: stock_picking_invoice_link/models/sale_order.py [hunk 8]
```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# © 2013-15 Agile Business Group sagl (<http://www.agilebg.com>)
+# © 2017 Jacques-Etienne Baudoux <je@bcim.be>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import api, models
```
Comment (grindtildeath) [inline stock_picking_invoice_link/models/sale_order.py:6]: from odoo

File: stock_picking_invoice_link/models/account_invoice.py [hunk 9]
```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# © 2013-15 Agile Business Group sagl (<http://www.agilebg.com>)
+# © 2015-2016 AvanzOSC
+# © 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# © 2017 Jacques-Etienne Baudoux <je@bcim.be>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import fields, models
```
Comment (grindtildeath) [inline stock_picking_invoice_link/models/account_invoice.py:8]: from odoo

**Review by grindtildeath — CHANGES_REQUESTED** [review]
Works great, just a few things to change though.

**Comment by eLBati** [comment]
Moving to https://github.com/OCA/stock-logistics-workflow/pull/347

@grindtildeath thanks, fixed there

**Pull request closed without merging**